docs/source/release_notes/release-notes.rst

```diff
@@ -55,6 +55,9 @@ develop
          - Fixed a bug that would cause console to error on any range request that used a column selection and had more than one batch of results.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1876>`__)
 
+    *    - |fixed|
```

Comment:
nit: I'd recommend `|new|` rather than `|fixed|` here, not having jar publication isn't a bug...